doc/classes/VisualShaderNodeFloatFunc.xml

```diff
@@ -1,81 +1,116 @@
+			Returns the arc-sine of the parameter. Translates to [code]asin(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_ACOS" value="4" enum="Function">
+			Returns the arc-cosine of the parameter. Translates to [code]acos(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_ATAN" value="5" enum="Function">
+			Returns the arc-tangent of the parameter. Translates to [code]atan(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_SINH" value="6" enum="Function">
+			Returns the hyperbolic sine of the parameter. Translates to [code]sinh(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_COSH" value="7" enum="Function">
+			Returns the hyperbolic cosine of the parameter. Translates to [code]cosh(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_TANH" value="8" enum="Function">
+			Returns the hyperbolic tangent of the parameter. Translates to [code]tanh(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_LOG" value="9" enum="Function">
+			Returns the natural logarithm of the parameter. Translates to [code]log(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_EXP" value="10" enum="Function">
+			Returns the natural exponentiation of the parameter. Translates to [code]exp(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_SQRT" value="11" enum="Function">
+			Returns the square root of the parameter. Translates to [code]sqrt(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_ABS" value="12" enum="Function">
+			Returns the absolute value of the parameter. Translates to [code]abs(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_SIGN" value="13" enum="Function">
+			Extracts the sign of the parameter. Translates to [code]sign(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_FLOOR" value="14" enum="Function">
+			Finds the nearest integer less than or equal to the parameter. Translates to [code]floor(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_ROUND" value="15" enum="Function">
+			Finds the nearest integer to the parameter. Translates to [code]round(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_CEIL" value="16" enum="Function">
+			Finds the nearest integer that is greater than or equal to the parameter. Translates to [code]ceil(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_FRAC" value="17" enum="Function">
+			Computes the fractional part of the argument. Translates to [code]fract(x)[/code] in the Godot Shader Language.
 		</constant>
 		<constant name="FUNC_SATURATE" value="18" enum="Function">
+			Clamps the value between [code]0.0[/code] and [code]1.0[/code] by [code]min(max(x, 0.0), 1.0)[/code] formula.
```

Comment:
Clamps the value between [code]0.0[/code] and [code]1.0[/code] using [code]min(max(x, 0.0), 1.0)[/code].